### PR TITLE
Re-Design

### PR DESCRIPTION
**Pull request by kentnorca**
Main Menu
>Image Changes

New Event
>Added Sliding Panel
>Change Button Style

### CODE REVIEW

**Comment by khrizthero24** [comment]
Change picture talaga. I dont want dat shit na.  Hahahahaha

**Comment by TeaRX27** [comment]
trying to merge 50% chance to break code

File: System/Program.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ static void Main()
         {
             Application.EnableVisualStyles();
             Application.SetCompatibleTextRenderingDefault(false);
-            Application.Run(new Login());
+            Application.Run(new New_Event());
```
Comment (TeaRX27) [inline System/Program.cs]:
```suggestion
            Application.Run(new Login());
```

**Review by TeaRX27 — CHANGES_REQUESTED** [review]
- Set pnlEventImage Size to 512, 470

**Comment by khrizthero24** [comment]
@kentnorca yung latest push mo sa branch mo wasak. @TeaRX27 pa rollback ng branch ni norca. salamat

**Comment by khrizthero24** [comment]
nevermind i may have just fucking saved the system. hahaha fucking merging

**Comment by kentnorca** [comment]
mag commit muna ako? bago mag pull?

**Pull request merged**